mods/crafting/init.lua

```diff
@@ -133,9 +133,9 @@ minetest.register_node("crafting:crafting_table_progressive", {
 api.recipes.add_implementor("crafting_table_progressive", "crafting:crafting_table_progressive")
 
 local recipe = api.recipes.add("multiblock_3_3",
-	{"crafting:crafting_table", "ws_core:wood", "crafting:crafting_table",
-	"ws_core:wood", "crafting:crafting_table", "ws_core:wood",
-	"crafting:crafting_table", "ws_core:wood", "crafting:crafting_table"},
+	{"crafting:crafting_table", "group:planks", "crafting:crafting_table",
+	"group:planks", "crafting:crafting_table", "group:planks",
+	"crafting:crafting_table", "group:planks", "crafting:crafting_table"},

```

Comment:
I'm not sure that this works, I'll fix activated multiblock code later today so it does

Reply:
Groups in multiblocks worked when I tried it.

Reply:
wow, interesting, I guess that's because it uses `api.item_satisfies`